src/WindowsStateTriggers/InteractionCapabilityHelper.cs

```diff
@@ -0,0 +1,72 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Windows.UI.ViewManagement;
+
+namespace WindowsStateTriggers
+{
+    /// <summary>
+    /// 
+    /// </summary>
+    public class InteractionCapabilityHelper
```

Comment:
You are right on marking the class as static. It is also declared as public so that devs can use the StatTrigger or directly calling the helper to allow more flexibility